src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodScopeAnonInner.java

```diff
@@ -1,59 +1,57 @@
-////////////////////////////////////////////////////////////////////////////////
-// Test case file for checkstyle.
-// Created: 2001
-////////////////////////////////////////////////////////////////////////////////
```

Comment:
Removal of these + addition of `/* Config` comment with an extra line might have prevented changing of violation line numbers in UT file :) No need to change now, but maybe remember for the future to save time.

Reply:
Resolved.